cob_twist_controller/include/cob_twist_controller/kinematic_extensions/kinematic_extension_base.h

```diff
@@ -51,7 +51,7 @@ class KinematicExtensionBase
 
         virtual bool initExtension() = 0;
         virtual KDL::Jacobian adjustJacobian(const KDL::Jacobian& jac_chain) = 0;
-        virtual JointStates adjustJointStates(const JointStates& joint_states) = 0;
+        virtual JointStates adjustJointStates(const JointStates& joint_states, const geometry_msgs::Pose& pose, const KDL::Twist& twist) = 0;
```

Comment:
base class has been changed and adapted in all files accordingly, even though `pose` and `twist` is only required in `kinematic_extension_dof`

@ipa-fxm is this fine for you?